### ISSUE TITLE
[Bug] npm installation fails

### ISSUE THREAD

**Issue by mnemanja** [STRML/react-grid-layout#440]
Hello,
Having trouble with the installation.

**Update:** When I've tried to install it in my React project then it worked without any problems. The problem occurs when I've tried to install it in an empty folder, alone.

After running the `npm install react-grid-layout` I get this message:
```
C:\www
+-- UNMET PEER DEPENDENCY react@^0.14.3 || ^15.0.0
+-- UNMET PEER DEPENDENCY react-dom@^0.14.3 || ^15.0.0
`-- react-grid-layout@0.13.9
  +-- UNMET PEER DEPENDENCY react@^0.14.0 || ^15.0.0
  `-- UNMET PEER DEPENDENCY react-dom@^0.14.0 || ^15.0.0

npm WARN enoent ENOENT: no such file or directory, open 'C:\www\package.json'
npm WARN react-grid-layout@0.13.9 requires a peer of react@^0.14.3 || ^15.0.0 but none was installed.
npm WARN react-grid-layout@0.13.9 requires a peer of react-dom@^0.14.3 || ^15.0.0 but none was installed.
npm WARN react-resizable@1.4.5 requires a peer of react@^0.14.0 || ^15.0.0 but none was installed.
npm WARN react-resizable@1.4.5 requires a peer of react-dom@^0.14.0 || ^15.0.0 but none was installed.
npm WARN www No description
npm WARN www No repository field.
npm WARN www No README data
npm WARN www No license field.
```

I've tried manually installing react and react-dom but to no avail.

I've tried with these node versions 6.5.0 and 5.7.0.

Windows 10.

**Comment by nikolas** [comment]
Yeah, it seems this package wasn't meant to be installed via `npm install`. I'm seeing the same error:

```
$ npm install react-grid-layout
/tmp/empty
├── UNMET PEER DEPENDENCY react@^0.14.3 || ^15.0.0
├── UNMET PEER DEPENDENCY react-dom@^0.14.3 || ^15.0.0
└─┬ react-grid-layout@0.13.9 
  ├── lodash.isequal@4.4.0 
  ├── UNMET PEER DEPENDENCY react@^0.14.0 || ^15.0.0
  ├── UNMET PEER DEPENDENCY react-dom@^0.14.0 || ^15.0.0
  ├─┬ react-draggable@2.2.3 
  │ └── classnames@2.2.5 
  └── react-resizable@1.4.5 

npm WARN enoent ENOENT: no such file or directory, open '/tmp/empty/package.json'
npm WARN react-grid-layout@0.13.9 requires a peer of react@^0.14.3 || ^15.0.0 but none was installed.
npm WARN react-grid-layout@0.13.9 requires a peer of react-dom@^0.14.3 || ^15.0.0 but none was installed.
npm WARN react-resizable@1.4.5 requires a peer of react@^0.14.0 || ^15.0.0 but none was installed.
npm WARN react-resizable@1.4.5 requires a peer of react-dom@^0.14.0 || ^15.0.0 but none was installed.
npm WARN empty No description
npm WARN empty No repository field.
npm WARN empty No README data
npm WARN empty No license field.

$ npm --version
4.0.3

$ node --version
v6.9.2

$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.1 LTS
Release:	16.04
Codename:	xenial
```

**Comment by STRML** [comment]
This is probably due to `prepublish` running after install (classic npm bug), will investigate.

**Comment by STRML** [comment]
The `prepublish` run is fixed in d5c6cb8baaf10ff689f0d5a5e18375b4eb886f1d - otherwise the peerDep errors are quite intentional - we don't want to risk installing an extra version of React - this is similar to how most React component packages work so this is not a bug.
